(DOCSP-23193): Update Flutter Manage Sync Session (#1979)
mongodb/docs-realm: source/sdk/flutter/sync/manage-sync-session.txt

- ``RealmResults query``: Required. A ``RealmResults`` object that you can create using the
  :ref:`Realm Query Language query <flutter-query-language>`.
- ``String name``: Optional. Name for the subscription that you can refer to.

> - ``bool update``:  Optional. Only use with named subscriptions. If ``update`` is true,
>   adding a subscription with an existing name will replace the existing query with the new query.

Author: ```suggestion
- ``bool update``:  Optional. When true,
  adding a subscription with an existing name will replace the existing query with the new query. 
  Only use with named subscriptions. 
```

Author: batching your suggestions for this sentence @nathan-contino-mongo

You can update a named subscription with a new query. To update a subscription's
query, open an update block with ``SubscriptionSet.update()``.

> In the callback function of the update block, pass the new query, the name of the subscription that you want to update,
> and ``update: true`` to ``MutableSubscriptionSet.add()``.

Author: ```suggestion
In the callback function of the update block, pass the following arguments to ``MutableSubscriptionSet.add()``:

- The new query
- The name of the subscription that you want to update
- ``update: true``
```

.. literalinclude:: /examples/generated/flutter/manage_sync_session_test.snippet.remove-all-subscriptions.dart
   :language: dart

>    :emphasize-lines: 2

Author: i like to stay consistent w my literal includes 😄
